src/content/docs/en/basics/astro-pages.mdx

```diff
@@ -96,10 +96,20 @@ Files with the `.html` file extension can be placed in the `src/pages/` director
 
 ## Custom 404 Error Page
 
-For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`.
+For a custom 404 error page, you can create a `404.astro` or `404.md` file in `src/pages`.
 
 This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
 
+## Custom 500 Error Page
+
+For a custom 500 error page to show for pages in your `hybrid` or `server` project that are [rendered on demand](/en/guides/server-side-rendering/#enable-on-demand-server-rendering), create the file `src/pages/500.astro`. This custom page is not available for prerendered pages.
```

Comment:
```suggestion
For a custom 500 error page to show for pages [rendered on demand](/en/guides/server-side-rendering/#enable-on-demand-server-rendering), create the file `src/pages/500.astro`. This custom page is not available for prerendered pages and can't be prerendered.
```
@sarah11918 wdyt?

Reply:
I'm in favour of the shorter wording! I would probably keep "that" as it helps with readability:

> for pages that are rendered on demand

It's just less dense and can be easier to read. But, LGTM! @florian-lefebvre